CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Change Log
 
+## 2.2
+
+This release is mostly a maintenance release which fixes bugs and regressions.
+It also unblocks significant future feature development.
+
+### Workaround removals
+
+- Remove old `force_bindless_texel_buffer` workaround with
+  a more correct and performant workaround.
+  Death Stranding and Cyberpunk 2077 (and probably other games as well) do the right thing by default without the hack now.
+- Remove old workaround for occlusion queries which was enabled for AC: Valhalla.
+  Improves performance in AC: Valhalla.
+
+#### Cyberpunk 2077 status
+From recent testing on our end, it is unknown at this time if `VK_VALVE_mutable_descriptor_type` is still required for
+Cyberpunk 2077. Manual testing hasn't been able to trigger a GPU hang.
+The memory allocation rewrite in 2.2 can plausibly work around some of the bugs that `VK_VALVE_mutable_descriptor_type` fixed by accident.
+The bugs in question could also have been fixed since release day, but we cannot prove this since the bug is completely random in nature.
+
+### Regression fixes
+
+- Fix regression in Horizon Zero Dawn for screen space reflections on water surfaces.
+
+### Stability fixes
+
+- Greatly improve stability on Polaris or older cards for certain titles.
+  Crashes which used to happen in Horizon Zero Dawn and Death Stranding seem to have disappeared
+  after the memory allocation rewrite.
```

Comment:
Revised; 

`Crashes in Horizon Zero Dawn and Death Stranding fixed with memory allocation rewrite.`

fixed can be interchanged with; resolved or improved